src/main/scala/nl/biopet/utils/tool/ToolDocumentation.scala

```diff
@@ -0,0 +1,163 @@
+    s"""
+       |<p>
+       |  <!-- Obscure e-mail address for spammers -->
+       |For any question related to this tool, please use the
+       |<a href='https://github.com/biopet/${toolName}/issues'>github issue tracker</a>
+       |or contact
+       |  <a href='http://sasc.lumc.nl/'>the SASC team</a> directly at: <a href='&#109;&#97;&#105;&#108;&#116;&#111;&#58;
+       | &#115;&#97;&#115;&#99;&#64;&#108;&#117;&#109;&#99;&#46;&#110;&#108;'>
+       |  &#115;&#97;&#115;&#99;&#64;&#108;&#117;&#109;&#99;&#46;&#110;&#108;</a>.
+       |</p>
+       |
+     """.stripMargin
+
+  // Universal text referring to BIOPET.
+  def aboutText: String =
+    """
+      |This tool is part of BIOPET tool suite that is developed at LUMC by [the SASC team](http://sasc.lumc.nl/).
+      |Each tool in the BIOPET tool suite is meant to offer a standalone function that can be used to perform a
+      |dedicate data analysis task or added as part of [BIOPET pipelines](http://biopet-docs.readthedocs.io/en/latest/).
+    """.stripMargin
+
+  // Universal installation text
+  def installationText: String =
+    s"""
+      |This tool requires Java 8 to be installed on your device. Download Java 8
+      |[here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html)
+      |or install via your distribution's package manager.
+      |
+      |Download the latest version of ${toolName} [here](https://github.com/biopet/${toolName}/releases/).
+      |To generate the usage run:
+      |
+      |    java -jar ${toolName}-version.jar --help
+      |
+    """.stripMargin
+
+  // Which chapters should be in the README
+  def readmeContents: List[(String,String)] = {
+    List(
+      (s"# ${toolName}",descriptionText),
+      ("# Documentation", documentationText),
+      ("# About", aboutText),
+      ("# Contact", contactText)
+
+    )
+  }
+
+  /**
+    * Generates a Markdown file from a list of chapters (heading, content) tuples.
+    * @param contents A list of (string, string) tuples, where the first string is the title and the other the content.
+    * @param outputFilename The output filename to which the markdown file is written.
+    */
+  def contentsToMarkdown(
+                        contents: List[(String, String)],
+                        outputFilename: String
+                   ): Unit = {
+    val outputFile = new File(outputFilename)
+    outputFile.getParentFile.mkdirs()
+    outputFile.createNewFile()
+    val fileWriter = new PrintWriter(outputFile)
+    for (entry <- contents){
+      fileWriter.write(
+        entry._1 + System.lineSeparator() +
+          System.lineSeparator() +
+        entry._2 + System.lineSeparator() +
+          System.lineSeparator()
+      )
+    }
+    fileWriter.close()
+  }
+
+  /**
+    * Converts a resource to a file
+    * @param resource Which resource
+    * @param output Filename for the output file
+    */
+  def resourceToFile(resource: String, output: String): Unit = {
```

Comment:
Not sure but this should be already in common-utils. If not this should be move to there.